### PR TITLE
Using context menu

### PR DESCRIPTION
**Pull request by wistcc**
These are the things implemented on this PR:
- Using the context menu
- Start typing only on the element where the context menu was clicked on
- Stop typing when the element loses focus

### CODE REVIEW

File: contentScript.js [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+chrome.runtime.onMessage.addListener(request => {
+    if (request === 'contextMenuClicked') {
+        writeOnElement(clickedElement)
+    }
+});
+
+/*
+    Move this code to a file called type.js
+    To do this we need to start using webpack since ES6 modules is not supported
+*/
```
Comment (nramirez) [inline contentScript.js:26]: what if we leave in this file forever 😅, I don't think this code will grow that much.

File: contentScript.js [hunk 2]
```diff
@@ -15,24 +42,26 @@ const cpm = wpm * 5;
...
-    writeChar(text[index]);
-    ++index;
-    console.log(window.getSelection());
+const writeOnElement = (element) => {
+    let index = 0;
+    interval = setInterval(() => {
+        console.log(text[index])
+        writeChar(element, text[index]);
+        ++index;
+        console.log(window.getSelection());
```
Comment (nramirez) [inline contentScript.js]: can you please remove these console logs? (L57), I don't think they're useful anymore if we want to debug something we can add it locally.

**Review by nramirez — APPROVED** [review]
:shipit:

File: contentScript.js (hunk 1 above)
Comment (wistcc) [reply contentScript.js:26]: Maybe.

**Pull request merged**